### PR TITLE
FB - Migrate your WordPress website to OVHcloud

### PR DESCRIPTION
**Pull request by fabienOVH**
Waiting for Guillaume F. + PU to review and give the "GO"

### CODE REVIEW

**Comment by GuiF008** [comment]
go to publish

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: "Tutoriel - Migrer son site web WordPress vers OVHcloud"
+excerpt: "Découvrez comment migrer votre site web WordPress et ses services associés vers OVHcloud"
+updated: 2024-06-26
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
updated: 2024-06-27
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Prérequis
+
+- Être connecté à l'interface d'administration de WordPress
+
+## En pratique
+
+### Étape 1 : sauvegarder les fichiers et la base de données de votre site web WordPress
+
+La première étape consiste à récupérer tous les fichiers relatifs à votre site web WordPress. Cela inclut les fichiers WordPress, ainsi que votre base de données. Consultez le guide vous expliquant comment [sauvegarder votre site WordPress](/pages/web_cloud/web_hosting/how_to_backup_your_wordpress).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
La première étape consiste à récupérer tous les fichiers relatifs à votre site web WordPress. Cela inclut les fichiers WordPress, ainsi que votre base de données. Consultez ensuite notre guide « [Sauvegarder votre site WordPress](/pages/web_cloud/web_hosting/how_to_backup_your_wordpress) ».
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+### Étape 2 : transférer votre site web WordPress vers OVHcloud
+
+Une fois les fichiers et la base de données de votre site web WordPress sauvegardés, vous devez les transférer vers votre hébergement web OVHcloud. Si vous ne possédez pas encore d'hébergement OVHcloud, suivez l'étape 1 du guide [Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
Une fois les fichiers et la base de données de votre site web WordPress sauvegardés, transférez-les vers votre hébergement web OVHcloud. Si vous ne possédez pas encore d'hébergement web OVHcloud, suivez l'étape 1 du guide « [Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh) ».
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Étape 2.1 : transférer les fichiers de votre site web WordPress
+
+> [!primary]
+>
+> Nous vous recommandons d'utiliser le logiciel [FileZilla](/pages/web_cloud/web_hosting/ftp_filezilla_user_guide) pour transférer vos fichiers WordPress vers votre hébergement.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
> Nous vous recommandons d'utiliser le logiciel [FileZilla](/pages/web_cloud/web_hosting/ftp_filezilla_user_guide) pour transférer vos fichiers WordPress vers votre hébergement web.
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+>
+
+Pour pouvoir transférer les fichiers relatifs à votre site web WordPress, vous devez d'abord vous [connecter à l'espace de stockage FTP de votre hébergement OVHcloud](/pages/web_cloud/web_hosting/ftp_connection).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
Pour transférer les fichiers relatifs à votre site web WordPress, connectez-vous d'abord à [l'espace de stockage FTP de votre hébergement web OVHcloud](/pages/web_cloud/web_hosting/ftp_connection).
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+Une fois connecté à l'espace de stockage FTP de votre hébergement OVHcloud, naviguez vers le répertoire racine « www » (ou dans un autre dossier racine que vous aurez préalablement créé). Si vos fichiers de sauvegarde sont compressés (zippés), décompressez-les dans un dossier vide sur votre ordinateur avant de téléverser vos fichiers dans le répertoire racine de votre hébergement web OVHcloud.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
Une fois connecté à l'espace de stockage FTP de votre hébergement web OVHcloud, naviguez vers le répertoire racine « www » (ou dans un autre dossier racine que vous aurez préalablement créé). Si vos fichiers de sauvegarde sont compressés (zippés), décompressez-les dans un dossier vide sur votre ordinateur avant de les téléverser dans le répertoire racine de votre hébergement web OVHcloud.
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Étape 2.1 : importer la base de données de votre site web WordPress
+
+Si vous n'en avez pas encore, [créez une nouvelle base de données](/pages/web_cloud/web_hosting/sql_create_database) puis [importez votre sauvegarde dans votre nouvelle base de données](/pages/web_cloud/web_hosting/sql_importing_mysql_database).
+
+> [!primary]
+>
+> OVHcloud propose des serveurs de base de données Web Cloud Databases. Si vous souhaitez utiliser cette offre avec votre site web, retrouvez l'ensemble de notre documentation sur ce produit sur notre page dédiée </products/web-cloud-clouddb>.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
> OVHcloud propose des serveurs de base de données Web Cloud Databases. Si vous souhaitez utiliser cette offre avec votre site web, retrouvez l'ensemble de notre documentation sur ce produit sur notre [page dédiée](/links/web/databases).
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+>
+
+### Étape 3 : mettre à jour les informations de la base de données
+
+Vous devez maintenant lier votre site web WordPress à votre base de données. Ces modifications sont à effectuer dans le fichier de configuration **« wp-config.php »**. Retrouvez l'ensemble des actions à effectuer en suivant les étapes du guide [modifier le mot de passe de la base de données d'un hébergement web](/pages/web_cloud/web_hosting/sql_change_password).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
Vous devez maintenant lier votre site web WordPress à votre base de données. Ces modifications sont à effectuer dans le fichier de configuration **« wp-config.php »**. Retrouvez l'ensemble des actions à effectuer en suivant les étapes du guide « [Modifier le mot de passe de la base de données d'un hébergement web](/pages/web_cloud/web_hosting/sql_change_password) ».
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+
+### Migrer d'autres services associés à votre site web WordPress
+
+Vous venez de migrer vos fichiers et votre base de données WordPress. Si vous souhaitez migrer d'autres services comme vos e-mails, votre nom de domaine ou encore vos zones DNS, suivez les étapes du guide [Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh) en vous reportant aux étapes qui concernent les services que vous souhaitez migrer. En effet, plusieurs des étapes auront déjà été effectuées dans ce guide.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
Vous venez de migrer vos fichiers et votre base de données WordPress. Si vous souhaitez migrer d'autres services comme vos e-mails, votre nom de domaine ou encore vos zones DNS, suivez les étapes du guide « [Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh) » en vous reportant aux étapes qui concernent les services que vous souhaitez migrer. En effet, plusieurs des étapes auront déjà été effectuées dans ce guide.
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+## Aller plus loin
+
+[Généralités sur les e-mails mutualisés](/pages/web_cloud/email_and_collaborative_solutions/mx_plan/email_generalities).
+
+[Migrer son site web et ses services associés vers OVHcloud](/pages/web_cloud/web_hosting/hosting_migrating_to_ovh).
+
+[Importation d’une base de données MySQL](/pages/web_cloud/web_hosting/sql_importing_mysql_database).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
[Importer une base de données MySQL](/pages/web_cloud/web_hosting/sql_importing_mysql_database).
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+
+[Gestion d’une base de données depuis un hébergement mutualisé](/pages/web_cloud/web_hosting/sql_create_database).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
[Créer une base de données sur son hébergement web](/pages/web_cloud/web_hosting/sql_create_database).
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+
+Pour des prestations spécialisées (référencement, développement, etc), contactez les [partenaires OVHcloud](/links/partner).
+
+Si vous souhaitez bénéficier d'une assistance à l'usage et à la configuration de vos solutions OVHcloud, nous vous proposons de consulter nos différentes [offres de support](/links/support).
+
+Échangez avec notre communauté d'utilisateurs sur <https://community.ovh.com/>.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
Échangez avec notre [communauté d'utilisateurs](/links/community).
```

**Review by Kazarus4 — APPROVED** [review]

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+> [!primary]
+>
+> Nous vous recommandons d'utiliser le logiciel [FileZilla](/pages/web_cloud/web_hosting/ftp_filezilla_user_guide) pour transférer vos fichiers WordPress vers votre hébergement web.
+>
+
+Pour transférer les fichiers relatifs à votre site web WordPress, connectez-vous d'abord à [l'espace de stockage FTP de votre hébergement web OVHcloud](/pages/web_cloud/web_hosting/ftp_connection).
+
+Une fois connecté à l'espace de stockage FTP de votre hébergement web OVHcloud, naviguez vers le répertoire racine « www » (ou dans un autre dossier racine que vous aurez préalablement créé). Si vos fichiers de sauvegarde sont compressés (zippés), décompressez-les dans un dossier vide sur votre ordinateur avant de les téléverser dans le répertoire racine de votre hébergement web OVHcloud.
+
+#### Étape 2.1 : importer la base de données de votre site web WordPress
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
#### Étape 2.2 : importer la base de données de votre site web WordPress
```

File: pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md [hunk 14]
```diff
@@ -0,0 +1,76 @@
+---
+title: "Tutoriel - Migrer son site web WordPress vers OVHcloud"
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/migrate_wordpress_website_to_ovh/guide.fr-fr.md]:
```suggestion
title: "Migrer son site web WordPress vers OVHcloud"
```

**Review by Kazarus4 — APPROVED** [review]

**Review by OVH-Team-Guides — COMMENTED** [review]
One last update to do.
The index also needs to be edited. The guide can be inserted into the `Tutorials - WordPress` category.

**Comment by Kazarus4** [comment]
Already insert into the index file

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @fabienOVH

**Pull request merged**